table.cc

```diff
@@ -1262,9 +1262,14 @@ future<std::unordered_map<sstring, table::snapshot_details>> table::get_snapshot
     });
 }
 
-future<> table::flush() {
+future<> table::flush(std::optional<db::replay_position> pos) {
+    if (pos && *pos < _flush_rp) {
```

Comment:
Shouldn't this be `<=`? So that two flushes at the same position get coalesced?

Maybe I'm mistaken about replay_position numbering.

Reply:
Technically I guess it should. Though it should not happen. This is to deal with out-of-ordering, in which case things will be non-equal, but potentially lower than threshold.